autoload/go/guru.vim

```diff
@@ -217,11 +213,19 @@ endfunc
 
 " run_guru runs the given guru argument
 function! s:run_guru(args) abort
+
+  let old_gopath = $GOPATH
+  let $GOPATH = go#path#Detect()
   if go#util#has_job()
-    return s:async_guru(a:args)
+    let abort = s:async_guru(a:args)
```

Comment:
I have no idea why I called it `abort`. I think I was distracted by the `abort` in the function signature. 

On closer examination... does the result need to be returned at all? No callers seem to use it. Can we just call an return nothing?